.circleci/config.yml

```diff
@@ -95,12 +91,10 @@ jobs:
           name: Run Accessibility Tests
           command: |
             pip install -U pip
-            pip install tox
+            pip install -r ./bin/require.pip
```

Comment:
Unless... maybe this is relying on `pip` calling Python 3? Since it appears there's some Python 3 bits down there.

Technically switching the Python version is as simple as changing this line: https://github.com/mozilla-services/screenshots/blob/master/Makefile#L94

It's possible some server tests would break if that was done. Trying it out myself, I note that we need to upgrade to `sarge==0.1.5.post0`, then `rm -r .venv`, and then `npm run test:server`.

But there's a bunch of small issues, and now my server isn't running for some dumb unrelated reason. I put my changes in #5070, but I can't complete that as I have some other work that is going to take priority for the rest of this week.